library/src/main/cpp/libenc/libenc.cc

```diff
@@ -225,15 +226,16 @@ YUV420_888toI420(uint8_t *src_y, jint y_stride,
             i++;
         }
     }
+
 //    ret = Android420ToI420(src_y, y_stride,
```

Comment:
Unused code, comment not used or in progress?

Reply:
Will remove it